roles/openshift_health_checker/test/search_journalctl_test.py

```diff
@@ -0,0 +1,190 @@
+import pytest
+import search_journalctl
+
+from time import time
+
+
+def canned_search_journalctl(get_log_input=None):
+    """Create a search_journalctl object with canned get_log_input method"""
+    module = search_journalctl
+    if get_log_input:
+        module._get_log_input = get_log_input
+    return module
+
+
+def get_timestamp_microseconds(modifier=0):
+    return str(time() + modifier).split(".")[0] + "000000"
```

Comment:
I dislike using this in tests because every time we run the tests we are using different test data.
This is an open door for a flake.